.gitignore

```diff
@@ -0,0 +1 @@
+.thoth.yaml
```

Comment:
This should be probably kept versioned.

Reply:
This is a local copy for I was using to test thamos cli, does this need to be versioned?

Reply:
I think it would be good to keep it versioned. It's a configuration file for Thoth that can change over time (or configuration options).

Reply:
Should I put the one pointing to `api.moc` or `khemenu`?

Reply:
`api.moc`

Reply:
Oh, I see now your point. The config file gets expanded during the build. In that case, it might be good not to have it in the repo for the build examples (might be confusing)? Or do we want to keep them there?

Reply:
That's why I thought of having them ignored would be a good idea, should we remove them?

Reply:
I don't have any strict opinion here. For s2i builds they could be removed not to confuse users. For examples that use CLI, they will need to be present.

Reply:
I think all of these are s2i examples, so we could remove them.

Reply:
OK, sounds good to me :+1: